### PR TITLE
Update css-handles

### PR DESCRIPTION
**Pull request by iaronaraujo**
#### What is the purpose of this pull request?
As the title says

#### How should this be manually tested?
[Workspsace](https://iaronaraujo--storecomponents.myvtex.com/)
[Als](https://iaronaraujo--alssports.myvtex.com/)
[tbb](https://iaronaraujo--tbb.myvtex.com/)

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/Menu.tsx [hunk 1]
```diff
@@ -48,14 +49,12 @@ const Menu: StorefrontFunctionComponent<MenuSchema> = ({
     )
   )
 
-  const classes = generateBlockClass(styles.menuContainer, blockClass)
-
   return (
     <LevelContext.Provider value={level + 1}>
       <MenuContext.Provider value={menuContext}>
           <nav>
```
Comment (Klynger) [inline react/Menu.tsx]: Missing handle?

File: react/MenuItem.tsx [hunk 2]
```diff
@@ -57,11 +58,10 @@ const MenuItem: StorefrontFunctionComponent<MenuItemSchema> = ({
    * in the future. */
   const submenuImplementation = useSubmenuImplementation()
   const isCollapsible = submenuImplementation === 'submenu.accordion'
-  const classes = generateBlockClass(styles.menuItem, blockClass)
 
   if (isCollapsible) {
     return (
-      <li className={classNames(classes, 'list')}>
+      <li className={classNames(handles.menuItem, 'list')}>
         <div
```
Comment (Klynger) [inline react/MenuItem.tsx]: Missing handle

File: react/Submenu.tsx [hunk 3]
```diff
@@ -34,12 +34,11 @@ const Submenu: StorefrontFunctionComponent<SubmenuProps> = ({
...
-  blockClass,
   orientation = Orientation.horizontal,
   paddingTop = 4,
   paddingBottom = 4,
 }) => {
-  const classes = generateBlockClass(styles.submenu, blockClass)
+  const handles = useCssHandles(CSS_HANDLES)
 
   return (
     <div className={`${width === '100%' ? '' : 'relative'}`}>
```
Comment (Klynger) [inline react/Submenu.tsx]: Missing handle

File: react/index.tsx [hunk 4]
```diff
@@ -33,199 +32,201 @@ interface Options {
...
-      </div>
-    )
+  if (!links.length) {
+    return null
   }
+  return (
+    <div className={`${handles.container} bg-base h2 c-muted-2 w-100 dn db-ns`}>
+      <Container>
+        <nav className="flex justify-between">
+          <div className="flex-grow pa3 flex items-center">
```
Comment (Klynger) [inline react/index.tsx]: Missing handles

File: docs/README.md [hunk 5]
```diff
@@ -139,19 +139,29 @@ The available `menu-item` block props are as follows:
...
+| `menuLinkNav`          |
+| `menuLinkDivLeft`      |
+| `menuLinkDivMiddle`    |
+| `menuLinkDivRight`     |
+| `renderLink`           |
+| `menuContainerNav`     |
+| `menuItemInnerDiv`     |
+| `submenuContainer`     |
+
+In order to apply CSS customizations in this and other blocks, follow the instructions given in the recipe on [Using CSS Handles for store customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization).
```
Comment (Klynger) [inline docs/README.md]: I think this text should be before the table and
```suggestion
In order to apply CSS customizations on this and other blocks, follow the instructions given in the recipe on [Using CSS Handles for store customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization). 
```

Comment (iaronaraujo) [reply docs/README.md]: I think it was in the end on Bia's template

**Pull request merged**